test/context-config.test.js

```diff
@@ -67,3 +67,55 @@ test('config', t => {
     t.deepEquals(JSON.parse(response.payload), { url: '/no-config', method: 'GET' })
   })
 })
+
+test('config with exposeHeadRoutes', t => {
+  t.plan(9)
+  const fastify = Fastify({ exposeHeadRoutes: true })
+
+  fastify.get('/get', {
+    schema: schema.schema,
+    config: Object.assign({}, schema.config)
```

Comment:
Why are you manually copying configs here?

Reply:
Not sure. I copy/pasted the previous test. I guess it's not needed anymore.